CMakeLists.txt

```diff
@@ -250,13 +255,16 @@ if(APPLE) # Apple specific installation of the app
   target_link_libraries(translateLocally PRIVATE ${LINK_LIBRARIES})
   set_target_properties(translateLocally PROPERTIES
     BUNDLE True
+    MACOSX_BUNDLE_BUNDLE_NAME ${CMAKE_PROJECT_NAME}
     MACOSX_BUNDLE_BUNDLE_VERSION ${PROJECT_VERSION_STRING_FULL}
     MACOSX_BUNDLE_SHORT_VERSION_STRING ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}
+    MACOSX_BUNDLE_GUI_IDENTIFIER "com.translatelocally.translateLocally"
     )
   # When on MacOS, we have a non-standard libarchive location because we get it via brew
   target_include_directories(translateLocally-bin PRIVATE ${LibArchive_INCLUDE_DIR})
   target_include_directories(translateLocally PRIVATE ${LibArchive_INCLUDE_DIR})
-  # produce .dmg
+  # produce .dmg This requires custom target due to https://gitlab.kitware.com/cmake/cmake/-/issues/16270
+  # use dist/macdmg.sh to sign and notarize
   include(macdeployqt)
```

Comment:
I think we should update README mac deployment instructions, since now it's no longer `cmake .. && make`, right?

Reply:
You're right, it would be kind of us to update the readme. Now it's either `make translateLocally && ../dist/macdmg.sh translateLocally.app` (with the right env variables, probably not worth documenting in detail) or `make translateLocally.dmg` (for your self-signed build) right?